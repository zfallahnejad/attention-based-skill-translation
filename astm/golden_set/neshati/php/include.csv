187606,include
476,include
2364629,include
138475,include
283366,include
345031,include
324584,include
454533,include
