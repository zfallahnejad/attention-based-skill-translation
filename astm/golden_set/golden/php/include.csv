187606,21,15,0.5833333333333334
476,19,22,0.4634146341463415
138475,10,8,0.5555555555555556
2364629,10,8,0.5555555555555556
345031,7,5,0.5833333333333334
324584,7,8,0.4666666666666667
454533,7,5,0.5833333333333334
283366,7,4,0.6363636363636364
